doc/classes/ProjectSettings.xml

```diff
@@ -2224,6 +2224,9 @@
 		<member name="physics/2d/run_on_separate_thread" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], the 2D physics server runs on a separate thread, making better use of multi-core CPUs. If [code]false[/code], the 2D physics server runs on the main thread. Running the physics server on a separate thread can increase performance, but restricts API access to only physics process.
 		</member>
+		<member name="physics/2d/run_on_separate_thread.editor" type="bool" setter="" getter="" default="false">
+			Editor override for [member physics/2d/run_on_separate_thread] to specify whether the editor's 2D physics server should run on a separate thread or not.
```

Comment:
Instead of repeating the this should explain _why_ this is overridden (something similar to what you've stated in this PR's description.

Reply:
Added: 
![image](https://github.com/user-attachments/assets/a08f4875-4b0c-4a51-8754-0f795dc90380)

This kind of inline with what you were thinking? 🤔

Reply:
Not too sure on what more to add to the 2D description, because from what I know, the 2D editor doesn't have any quirks (high refresh rate issues) that aren't explained in the above `physics/2d/run_on_separate_thread`. Any thing more specific you had in mind?